master/buildbot/status/web/build.py

```diff
@@ -290,8 +291,22 @@ def content(self, req, cxt):
         cxt['build_url'] = path_to_build(req, b, False)
         cxt['slave_debug_url'] = self.getBuildmaster(req).config.slave_debug_url
         cxt['customBuildUrls'] = b.getCustomUrls()
+        cxt['build_chain_top_build_url'] = None
         codebases_arg = cxt['codebases_arg'] = getCodebasesArg(request=req)
 
+        build_chain_id = b.buildChainID
```

Comment:
I'd get rid of this local variable, or at least use it inside your if context (there are still a couple places using `b.buildChainID` in there)